### PR TITLE
remove unnecessary labels from equipment effects REs A-E

### PR DESCRIPTION
**Pull request by mysurvive**
Removed unnecessary labels from the following equipment effects REs (each variant):
-aeon stone black
-aeon stone orange
-antidote
-antiplague
-applereed mutagen
-bloodhound mask
-boulderhead bock
-bravo's brew
-breastplate of command
-Brewer's Regret
-Bronze Bull Pendant
-clandestine cloak
-cognitive mutagen
-Curse of Potent Poison
-Crimson Fulcrum Lens
-Deadweight Snare
-Diplomat's Badge
-Dragon's Blood Pudding
-Dragonfly Potion
-Drakeheart mutagen
-Dueling Cape
-Eagle Eye Elixir
-Earplugs
-Ebon Fulcrum Lens
-Elixir Of Life

### CODE REVIEW

**Comment by stwlam** [comment]
Are you sure all these are unnecessary? Some indicate the condition under which they apply.

**Comment by mysurvive** [comment]
@TikaelSol I spoke to Shark on discord about this. Can you take a look and if this isn't correct, can you shoot me a message on Discord when you're free so I can get some clarification on label removal?

**Comment by TikaelSol** [comment]
This is the current standard, we don't add labels to REs unless necessary and when necessary we localize them. None of these appear necessary to me.

File: packs/data/equipment-effects.db/effect-cognitive-mutagen-lesser.json [hunk 1]
```diff
@@ -19,28 +19,24 @@
...
                 "selector": "occultism",
                 "type": "item",
                 "value": 1
             },
             {
                 "key": "FlatModifier",
-                "label": "Cognitive Mutagen (Lesser)",
                 "selector": "society",
                 "type": "item",
                 "value": 1
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-cognitive-mutagen-lesser.json:42]: We'll have to circle back to this to condense these, selectors can be an array now and that will let us make this much cleaner.

File: packs/data/equipment-effects.db/effect-drakeheart-mutagen-greater.json [hunk 2]
```diff
@@ -24,35 +24,28 @@
...
-                "type": "untyped",
                 "value": -1
             },
             {
                 "key": "FlatModifier",
-                "label": "Drakeheart Mutagen (Greater)",
                 "selector": "reflex",
-                "type": "untyped",
                 "value": -1
             },
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-drakeheart-mutagen-greater.json:46]: Same here with condensing these two together.

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-greater.json [hunk 3]
```diff
@@ -18,14 +19,12 @@
...
                 "selector": "perception",
                 "type": "item",
                 "value": 3
             },
             {
                 "key": "FlatModifier",
-                "label": "Eagle Eye Elixir Greater (secret doors & traps)",
                 "predicate": [
                     "secret doors and traps"
                 ],
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-greater.json:30]:
```suggestion
                "label": "PF2E.SpecificRule.Equipment.EagleEyeElixir.Greater.SecretLabel",
                "slug": "eagle-eye-elixir-greater-secrets",
                "predicate": [
                    {
                        "or": ["secret-door", "trap"]
                    }
                ],
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-lesser.json [hunk 4]
```diff
@@ -18,14 +19,12 @@
         "rules": [
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-lesser.json:21]:
```suggestion
                "key": "FlatModifier",
                "slug": "eagle-eye-elixir-lesser",
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-lesser.json [hunk 5]
```diff
@@ -18,14 +19,12 @@
...
-                "label": "Eagle Eye Elixir (Lesser)",
                 "selector": "perception",
                 "type": "item",
                 "value": 1
             },
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-lesser.json:27]:
```suggestion
                "key": "FlatModifier",
                "label": "PF2E.SpecificRule.Equipment.EagleEyeElixir.Lesser.SecretLabel",
                "slug": "eagle-eye-elixir-lesser-secrets",
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-major.json [hunk 6]
```diff
@@ -18,14 +19,12 @@
         "rules": [
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-major.json:21]:
```suggestion
                "key": "FlatModifier",
                "slug": "eagle-eye-elixir-major",
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-major.json [hunk 7]
```diff
@@ -18,14 +19,12 @@
...
-                "label": "Eagle Eye Elixir (Major)",
                 "selector": "perception",
                 "type": "item",
                 "value": 3
             },
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-major.json:27]:
```suggestion
                "key": "FlatModifier",
                "label": "PF2E.SpecificRule.Equipment.EagleEyeElixir.Major.SecretLabel",
                "slug": "eagle-eye-elixir-major-secrets",
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-moderate.json [hunk 8]
```diff
@@ -18,14 +19,12 @@
         "rules": [
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-moderate.json:21]:
```suggestion
                "key": "FlatModifier",
                "slug": "eagle-eye-elixir-moderate",
```

File: packs/data/equipment-effects.db/effect-eagle-eye-elixir-moderate.json [hunk 9]
```diff
@@ -18,14 +19,12 @@
...
-                "label": "Eagle Eye Elixir (Moderate)",
                 "selector": "perception",
                 "type": "item",
                 "value": 2
             },
             {
                 "key": "FlatModifier",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-eagle-eye-elixir-moderate.json:27]:
```suggestion
                "key": "FlatModifier",
                "label": "PF2E.SpecificRule.Equipment.EagleEyeElixir.Moderate.SecretLabel",
                "slug": "eagle-eye-elixir-moderate-secrets",
```

File: packs/data/equipment-effects.db/effect-ebon-fulcrum-lens-2-action.json [hunk 10]
```diff
@@ -19,42 +19,36 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Ebon Fulcrum Lens Reaction",
                 "selector": "attack",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-ebon-fulcrum-lens-2-action.json:22]: More selector consolidation notes

File: packs/data/equipment-effects.db/effect-ebon-fulcrum-lens-reaction.json [hunk 11]
```diff
@@ -19,42 +19,36 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Ebon Fulcrum Lens Reaction",
                 "selector": "attack",
```
Comment (TikaelSol) [inline packs/data/equipment-effects.db/effect-ebon-fulcrum-lens-reaction.json:22]: ibid

**Comment by TikaelSol** [comment]
Ok, spoke with shark and figured out what he meant. We need to be careful about multiple flat modifiers to the same selector.

**Pull request merged**